README.md

```diff
@@ -1,2 +1,29 @@
-# mv-oop.rb
-An educational project to practice OOP in Ruby
+# School Library Exercise
```

Comment:
- [ ] According to the [Microverse Readme Template](https://github.com/microverseinc/readme-template), the readme file is missing the "Built with" and "Acknowledgments" sections please, kindly add the so we can meet this requirement.
- [ ] Please, kindly provide the steps to clone and run the code in your "Getting Started" section.

That way, your project will be well documented 💪

Reply:
I think I mentioned that this project needs Ruby but ok, I will add specific section for build with.

Reply:
@Meltrust There is nothing to run in this project so far. What do you mean by providing steps to run?

Reply:
> @Meltrust There is nothing to run in this project so far. What do you mean by providing steps to run?

Hey @ahangarha, just the steps to set up the project should suffice, like how to clone it, etc.

Reply:
I will add this but this is really not a good thing to put in repository.
One can download it as zip file. There is no need to clone this project. If someone doesn't know he should clone, he doesn't have git in his system. So I should mention how to install git and run git clone! Of course not.

I don't think this project needs anything more. I will add it but I want to share my unhappiness with this request. It is not a good reason for blocking the merge.